### PR TITLE
E-700 refactor: remove web3 usage and replace to user address

### PR DESCRIPTION
**Pull request by dudiq**

### CODE REVIEW

**Comment by dudiq** [comment]
@lamartire ready for review

File: src/components/screen/AppLayout/NavSidebar.vue [hunk 1]
```diff
@@ -36,39 +32,22 @@
             </v-super-menu-control>
           </router-link>
         </li>
+        <!--        <li>-->
```
Comment (lamartire) [inline src/components/screen/AppLayout/NavSidebar.vue]: Remove useless comments

File: src/components/screen/AppLayout/QuickActions.vue [hunk 2]
```diff
@@ -8,20 +8,13 @@
...
-        <span class="caption">Provider</span>
-      </router-link>
-      <router-link
-        :to="{ name: 'CustomElement' }"
-        active-class="is-active"
-        class="navbar-item button"
-      >
-        <span class="caption">Custom Element</span>
-      </router-link>
+      <!--      <router-link-->
```
Comment (lamartire) [inline src/components/screen/AppLayout/QuickActions.vue]: Remove useless comments

**Comment by ifiri** [comment]
@dudiq web3 page is removed, but can't login. Maybe I doing something wrong?
image

**Comment by kazurus** [comment]
@dudiq please, remove mount/unmount

File: src/components/screen/AppLayout/QuickActions.vue (hunk 2 above)
Comment (dudiq) [reply src/components/screen/AppLayout/QuickActions.vue]: @kazurus was asked leave this comments for a while

File: src/components/screen/CustomElement/CustomElement.vue [hunk 3]
```diff
@@ -44,14 +44,23 @@ export default {
...
     node.addEventListener('close', () => {
-      console.log('dialog is close');
+      console.log('close');
     });
   },
 
   components: { FormField, VClientId, VContent, VButton },
 };
 </script>
+
```
Comment (dudiq) [inline src/components/screen/CustomElement/CustomElement.vue]: lost and found

**Comment by dudiq** [comment]
@ifiri check please again, but don't forget about call `yarn`, because deps was updated
@kazurus `mount/unmount` buttons was removed

**Comment by ifiri** [comment]
@kazurus approved, seems to all is ok

**Pull request merged**